packages/ui/src/pages/AdminPage.vue

```diff
@@ -0,0 +1,9 @@
+<script setup lang="ts"></script>
```

Comment:
Redirect (to home or something) if the user is not admin (see login page for example, which redirects if the user IS logged in)

Reply:
i added a check if user is not admin then redirect to home. i was testing it out on the ui and something that came up was if you're not logged in then if you go to /admin the page is blank. should i add a redirect for this or will we do it in the future for all pages?

Reply:
oh, that's because you're trying to access `isAdmin` on `authStore.currentUser`, which is `null`. typescript should yell at you for this, but the type is wrong - in `stores/auth.ts`, change `useStorage<IUser>` to `useStorage<IUser | null>`. Then ts will catch the error in AdminPage and make you change it to `authStore.currentUser?.isAdmin`